source/logs.txt

```diff
@@ -12,10 +12,10 @@ Retrieve Shell Logs
    :depth: 1
    :class: singlecol
 
-|mdb-shell| uses `ndjson <http://ndjson.org/>`__ to store session logs.
-Starting in :binary:`mongosh` version 1.0.5, the log |mdb-shell|
-format is updated to match the MongoDB server log format. For details,
-see :ref:`log-messages-ref`. 
+|mdb-shell| uses `ndjson <https://github.com/ndjson/ndjson-spec>`__ to
```

Comment:
```suggestion
|mdb-shell| uses `newline-delimited JSON <https://github.com/ndjson/ndjson-spec>`__ to
```

Just a suggestionm, but if we spell out what `ndjson` stands for, this format almost becomes self-explanatory anyway 🙂

Reply:
Made this change! Thank you @addaleax !